README.md

```diff
@@ -1 +1,66 @@
 # wizard-additional-playing-cards
+
+## Tornado | 14 | blau
+
+- Nach dem Stich wird in die andere Richtung gespielt
+- Bis zum Ende der Runde
+
+## Katapult | 14 | rot
+
+- Alle ziehen eine Karte
```

Comment:
Was machen in dem Letzten Stich/ oder, wenn nicht genug Karten für alle da sind.

Reply:
Wann muss man das ansagen/ spielen? Am Anfang einer Runde, oder während einer Runde?

Reply:
Man muss das gar nicht absagen. Das passiert wenn die Karte gelegt wird.

Reply:
> Man muss das gar nicht absagen. Das passiert wenn die Karte gelegt wird.

Dann sollte das mit in der Beschreibung stehen 🤫

Reply:
> Alternativ könnte man sich merken dass die Karte gespielt wurde und die Karten neu verteilen wenn der Frosch gespielt wird. Die Karte ist nämlich das Gegenstück zum Frosch frog. Ist aber vllcht zu kompliziert.

Das ist zu kompliziert/ Fehleranfällig. Wenn dann sollte das wie ein vanilla Werwolf, oder auch Jongleur gehandhabt werden, mit anderen Worten, die Karte macht gar nichts.